aries-site/src/pages/templates/forms.mdx

```diff
@@ -46,9 +46,8 @@ Complicated processes where the user might need guidance at various stages, or w
 
 - Ask only for the required information. Keep the fields to a minimum to allow the user to finish their task as quickly as possible.
 - List only the most common field requirements below the relevant field.
-- Use asterisks for required fields when the form has more than two required fields and/or has optional fields.
```

Comment:
I feel like this should stay. For example, what if a for had 1 required field and 1 optional field -- wouldn't we want to mark the required field to differentiate it?

Reply:
I agree with this. I think we should keep this section. We could just remove, " and/or has optional fields."

Reply:
I am saying we should not remove "and/or has optional fields" because of the example I listed in my comment.